hrm/department_app/models/employee_model.py

```diff
@@ -0,0 +1,39 @@
+"""Django models"""
+from django.db import models
+from department_app.models import department_models
+from djmoney.models.fields import MoneyField
+
+class Employee(models.Model):
+    """Employees"""
+    name = models.CharField('Name',max_length=50,primary_key=True)
+
+    department = models.ForeignKey(department_models.Department,on_delete=models.SET_NULL,null=True)
```

Comment:
I disagree that the department field can be null or empty. We cannot hire an employee without assigning him to any department.